src/org/openjdk/asmtools/common/Environment.java

```diff
@@ -79,31 +76,14 @@ public void setStrictWarnings(boolean strictWarnings) {
 
     public String getSimpleInputFileName() { return toolLogger.getSimpleInputFileName(); }
 
-    public String getInputFileName() { return inputFileName; }
+    public ToolInput getInputFile() { return inputFileName; }
 
     /**
      * @return DataInputStream or null if the method can't read a file
      */
     protected DataInputStream getDataInputStream() throws URISyntaxException, IOException {
         Objects.requireNonNull(this.inputFileName, "Input file name should be defined.");
-        try {
```

Comment:
Does this message reflect to introduced changes?

Reply:
you are right. It is weird as I left it